File: console-group/users/000002/BAI1.cpp
#include <iostream>
#include <fstream>
#include <algorithm>

using namespace std;

int main() {
  int a, b;
  cin >> a >> b;
  
  int gdc = __gcd(a, b);
  int lcm = (a * b) / gdc;
  
  cout << gdc << " " << lcm;
  return 0;
}

File: console-group/results/2022-11-13_14-02-40/000001/BAI1/TEST05/BAI1.py
def gdc(x, y):
  if y == 0:
    return x
  return gdc(y, x % y)


def lcm(x, y):
  return int((x * y) / gdc(x, y))


a , b = [int(x) for x in input().split()]

print("{} {}".format(gdc(a, b), lcm(a, b)))
